src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.eventCallback);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.eventCallback);
+  }
+
+  eventCallback = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render(): React.ReactNode {
+    return (
+      <div className="App">
+        <p className="App__message">
+          { this.state.pressedKey === ''
+            ? 'Nothing was pressed yet'
+            : `The last pressed key is [${this.state.pressedKey}]` }
```

Comment:
1. Simplify  condition
2. Handle the truth case first
```suggestion
          { this.state.pressedKey
            ? `The last pressed key is [${this.state.pressedKey}]` 
            :  'Nothing was pressed yet'}
```